03-nlp/languages.http

```diff
@@ -0,0 +1,255 @@
+Reference: https://learn.microsoft.com/en-gb/rest/api/language/text-analysis-runtime/analyze-text?view=rest-language-2023-04-01&tabs=HTTP
+
+### Language Detection
+POST {{ language endpoint }} /language/:analyze-text?api-version=2023-04-01 HTTP/1.1
```

Comment:
- Comments in `.http` files should only have one `#`. The three `###` are how the REST Client extension separates REST requests. One `#` is for comments.
- Please use camelCase for the variables, so the language endpoint should be `languageEndpoint` which is consistent to the others we have in the repo.

Reply:
Remove the space